src/state_machine.zig

```diff
@@ -1837,7 +1834,7 @@ test "create/lookup/rollback transfers" {
                 .id = 1,
                 .debit_account_id = 100,
                 .credit_account_id = 200,
-                .timeout = 1,
+                .timeout = (std.math.maxInt(u64) - timestamp) + 1,
                 .ledger = 0,
```

Comment:
Are these intended to trigger overflows_timeout?

Reply:
The idea with these tests is that they are checking the error code precedence, at the same time as they check the different error cases. In this particular case, we are checking that a `timeout_reserved_for_pending_transfer` error takes precedence over the `overflows_timeout` error (i.e. the former is the result) even though both are actually applicable to this transfer.

Reply:
Oh, oops, I see the comment way up at the top :)